src/main/java/klerer/rijks/RijksServiceFactory.java

```diff
@@ -0,0 +1,17 @@
+package klerer.rijks;
+
+import retrofit2.Retrofit;
+import retrofit2.adapter.rxjava3.RxJava3CallAdapterFactory;
+import retrofit2.converter.gson.GsonConverterFactory;
+
+public class RijksServiceFactory {
+    public RijksService getService() {
+        Retrofit retrofit = new Retrofit.Builder()
+                .baseUrl("https://www.rijksmuseum.nl/en")
```

Comment:
This isn't right. See the errors and look back at the url in their documentation.